### DISCUSSION TITLE
可以同时发布两种图标的版本吗

### DISCUSSION THREAD

**Discussion by Zhenghao-Wen** [gkd-kit/gkd#671] [Ideas]
我家**鸽鸽**太显眼了，希望作者尊重一些用户的需求

**Comment by lisonge** [comment]
你可以自己编译发布

**Reply by Zhenghao-Wen** [reply]
我已经在奇妙应用社区发布

**Reply by Zhenghao-Wen** [reply]
只需要反res资源混淆后
把/res目录下的
ic_launcher_background.xml和
ic_launcher_foreground.xml两个文件
替换成1.8.0beta7的文件即可

**Reply by lisonge** [reply]
那你应该在用户进入应用时给出弹窗里明确告知用户这是修改版，而不是官方正式版

否则用户使用你的修改版本，出了 BUG，反而来找我反馈，我修复不了，这不是浪费我时间吗？

**Reply by lisonge** [reply]
增加了一些说明 https://github.com/orgs/gkd-kit/discussions/679

**Reply by lisonge** [reply]
你好，我已经在 奇妙应用 看到有人发布了修改版

首先开源允许自由修改，但不是修改后随意发布，显然上述的修改版没有按照这个规范修改

这显然会给用户和开发者造成不必要的麻烦的困惑，希望能按要求修改后重新发布

如果在一段时间内没有修改，我将向奇妙应用举报投诉这个修改版

**Reply by Zhenghao-Wen** [reply]
好的，感谢指正。🙏🏻
